### PR TITLE
PMM-6375 sign out

### PR DESCRIPTION
**Pull request by nicolalamacchia**
[PMM-6375](https://jira.percona.com/browse/PMM-6375)

https://github.com/Percona-Lab/pmm-submodules/pull/1059

### CODE REVIEW

**Review by lunaticusgreen — APPROVED** [review]
LGTM

File: pmm-app/package.json [hunk 1]
```diff
@@ -40,6 +40,8 @@
     "@grafana/data": "^7.1.3",
     "@grafana/runtime": "^7.1.3",
     "@grafana/ui": "^7.1.3",
+    "@emotion/core": "^10.0.35",
+    "@types/jest": "^26.0.10",
```
Comment (atymchuk) [inline pmm-app/package.json:44]: This is devDependency

**Review by tiagomotasantos — APPROVED** [review]

**Comment by atymchuk** [comment]
@nicolalamacchia Where did the code disappear ?:(? My comments go to dev/null )

File: pmm-app/src/shared/components/helpers/validators.ts [hunk 2]
```diff
@@ -46,13 +53,22 @@ const validators = {
...
+
+    return 'Password must contain at least 8 characters, including UPPER/lower case and numbers, must not contain username';
+  },
   required: (value) => (value ? undefined : 'Required field'),
 
   requiredTrue: (value: boolean) => (value === true ? undefined : 'Required field'),
 
-  compose: (...validators) => (value) => validators.reduce(
-    (error, validator) => error || validator(value), undefined
+  compose: (...validators) => (value, values) => validators.reduce(
```
[the remark this answers is not in the document]
Comment (atymchuk) [reply pmm-app/src/shared/components/helpers/validators.ts:85]: Solved in a different branch, no worries.

**Review by atymchuk — CHANGES_REQUESTED** [review]

File: pmm-app/package.json [hunk 3]
```diff
@@ -40,6 +40,8 @@
     "@grafana/data": "^7.1.3",
     "@grafana/runtime": "^7.1.3",
     "@grafana/ui": "^7.1.3",
+    "@emotion/core": "^10.0.35",
```
[the remark this answers is not in the document]
Comment (nicolalamacchia) [reply pmm-app/package.json:43]: Removing this break the build, this solves some TypeScript checks apparently, so I'm keeping it for now (it's outside the scope of this PR)

**Review by atymchuk — APPROVED** [review]

**Pull request merged**